common/src/main/java/org/apache/falcon/metadata/InstanceRelationshipGraphBuilder.java

```diff
@@ -256,34 +256,47 @@ public void addEvictedInstance(WorkflowExecutionContext context) throws FalconEx
         }
     }
 
-
     public void addImportedInstance(WorkflowExecutionContext context) throws FalconException {
+        addImportExportInstanceHelper(context, RelationshipLabel.DATASOURCE_IMPORT_EDGE);
+    }
 
-        String feedName = context.getOutputFeedNames();
-        String feedInstanceDataPath = context.getOutputFeedInstancePaths();
+    public void addExportedInstance(WorkflowExecutionContext context) throws FalconException {
+        addImportExportInstanceHelper(context, RelationshipLabel.DATASOURCE_EXPORT_EDGE);
+    }
+
+    private void addImportExportInstanceHelper(WorkflowExecutionContext context,
+                                               RelationshipLabel label) throws FalconException {
+        String feedName = (label == RelationshipLabel.DATASOURCE_IMPORT_EDGE)
+                ? context.getOutputFeedNames() : context.getInputFeedNames();
```

Comment:
IMPORT -> gets data in to HDFS and so, it needs output path or partitions generated by Coordinator to write to and context.getOutputFeedNames()